### PR TITLE
Update travis

### PR DESCRIPTION
**Pull request by orangejulius**
Same changes as in other PRs like https://github.com/pelias/openstreetmap/pull/149

### CODE REVIEW

**Comment by trescube** [comment]
:shipit:

File: package.json [hunk 1]
```diff
@@ -7,14 +7,15 @@
   "license": "MIT",
   "main": "index.js",
   "scripts": {
-    "start": "node index.js",
-    "test": "npm run unit",
-    "unit": "./bin/units",
+    "audit": "npm shrinkwrap; node node_modules/nsp/bin/nspCLI.js audit-shrinkwrap; rm npm-shrinkwrap.json;",
```
Comment (dianashk) [inline package.json]: there are now quite a few scripts in each module. would be great to document this somewhere in a common place to explain what each one is for.

**Comment by dianashk** [comment]
LGTM

File: package.json (hunk 1 above)
Comment (orangejulius) [inline package.json]: good idea!

**Pull request merged**